README.md

```diff
@@ -9,6 +9,15 @@ Cloudfuse supports clouds with an S3 or Azure interface.
 ## Installation
 Download the provided installation packages for your preferred operating system.  -- COMING SOON!
 
+### from Windows .exe installer

```

Comment:
I think this sub-heading should just be "Windows."
The language here should be as plain and brief as possible.